acceptance/import_installation_test.go

```diff
@@ -0,0 +1,142 @@
+package acceptance
+
+import (
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"net/http/httptest"
+	"net/http/httputil"
+	"os"
+	"os/exec"
+	"path/filepath"
+
+	"github.com/onsi/gomega/gbytes"
+	"github.com/onsi/gomega/gexec"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+var _ = Describe("import-installation command", func() {
+	var (
+		installation string
+		passphrase   string
+		content      *os.File
+		server       *httptest.Server
+	)
+
+	BeforeEach(func() {
+		var err error
+		content, err = ioutil.TempFile("", "cool_name.com")
+		Expect(err).NotTo(HaveOccurred())
+
+		_, err = content.WriteString("content so validation does not fail")
+		Expect(err).NotTo(HaveOccurred())
+
+		server = httptest.NewTLSServer(http.HandlerFunc(func(w http.ResponseWriter, req *http.Request) {
+			var responseString string
+			w.Header().Set("Content-Type", "application/json")
+
+			switch req.URL.Path {
+			case "/api/v0/diagnostic_report":
+				responseString = "{}"
```

Comment:
Why is there a diagnostic call on this test when no call is made to this endpoint?